src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityAPI.kt

```diff
@@ -162,10 +162,12 @@ object HoppityAPI {
     // If there is a time since lastHoppityCallAccept, we can assume this is an abiphone call
     private fun getBoughtType(): HoppityEggType = if (lastHoppityCallAccept != null) BOUGHT_ABIPHONE else BOUGHT
 
-    fun isHoppityEvent() = (SkyblockSeason.currentSeason == SkyblockSeason.SPRING || SkyHanniMod.feature.dev.debug.alwaysHoppitys)
-    fun getEventEndMark(): SimpleTimeMark? = if (isHoppityEvent()) {
-        SkyBlockTime.fromSbYearAndMonth(SkyBlockTime.now().year, 3).asTimeMark()
-    } else null
+    fun isHoppityEvent() = (SkyblockSeason.SPRING.isSeason() || SkyHanniMod.feature.dev.debug.alwaysHoppitys)
+    fun getEventEndMark(): SimpleTimeMark? = if (isHoppityEvent()) SkyBlockTime.now().year.getEventEndMark() else null
+    fun Int.getEventEndMark(): SimpleTimeMark =
+        SkyBlockTime.fromSeason(this, SkyblockSeason.SUMMER, SkyblockSeason.SkyblockSeasonModifier.EARLY).asTimeMark()
+    fun Int.getEventStartMark(): SimpleTimeMark =
```

Comment:
functions that extend generic classes should be private. in this case, please dont use extent classes in APIs. (we dont want to have the suggestion to call hoppity event functions on int variables in the whole project)